code/modules/tgui/tgui_alert.dm

```diff
@@ -33,8 +35,10 @@
 	if(length(items) > 3)
 		log_tgui(user, "Error: TGUI Alert initiated with too many items. Use a list.", "TguiAlert")
 		return tgui_input_list(user, message, title, items, timeout, autofocus)
-	var/datum/tgui_modal/alert = new(user, message, title, items, timeout, autofocus)
+	var/datum/tgui_modal/alert = new(user, message, title, items, timeout, autofocus, content_window)
 	alert.ui_interact(user)
+	if (!do_wait)
+		return
```

Comment:
cleanup?

Reply:
Ref stops existing after closing the popup
![image](https://github.com/goonstation/goonstation/assets/6396368/87cbfb95-3e10-4f5c-9e94-313718015c9e)